conftest.py

```diff
@@ -0,0 +1,5 @@
+"""A pytest ``conftest.py`` file."""
+
+import os
+
+collect_ignore = [os.path.join('src', 'qmflows', 'settings.py')]
```

Comment:
Ignore `.../settings.py`, as it has issues failing the doctests due to inheriting a few faulty `plams.Settings`-related docstring.

Reply:
> Ignore `.../settings.py`, as it has issues failing the doctests due to inheriting a few faulty `plams.Settings`-related docstring.

Thanks for let me know